e2e_playwright/st_vega_lite_chart_test.py

```diff
@@ -66,7 +64,6 @@ def test_vega_lite_chart_same_plot_different_ways(
 ):
     """Tests that it displays the same plot in different ways"""
     vega_lite_charts = themed_app.get_by_test_id("stArrowVegaLiteChart")
-    expect(vega_lite_charts).to_have_count(13)
```

Comment:
Its better to keep this in for all cases that don't have another `expect` before the `assert_snapshot` because it potentially reduces flakiness.

`expect(vega_lite_charts).to_have_count(13)` makes sure that all the charts are in the DOM before it proceeds to do the snapshots.

Reply:
Ah I was not aware. let me add a comment to that because I got confused about this haha